src/plugins/discover/public/application/components/context_app/context_app_legacy.tsx

```diff
@@ -131,16 +175,20 @@ export function ContextAppLegacy(renderProps: ContextAppProps) {
       ) : (
         <div>
           <TopNavMenu {...getNavBarProps()} />
-          <EuiPage>
-            <EuiPageContent paddingSize="s" className="dscCxtAppContent">
+          <EuiPage className={isLegacy ? '' : 'dscSurrDocsPage'}>
```

Comment:
dear @elastic/kibana-design , introducing `EuiDataGrid` will also change layout in the Discover context view, just to inform you, that you could have an early look here for feedback, many thx!